src/Symfony/Component/Translation/Resources/bin/translation-status.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+$usageInstructions = <<<END
+
+  Usage instructions
+  ---------------------------------------------------------------------
+
+  $ cs symfony-code-root-directory/
+
+  # show the translation status of all locales
+  $ php translation-status.php
```

Comment:
I might be missing something here, but this file isn't in the root directory of the project, so should the full path not be added here?